### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by JerryIshihara** [JerryIshihara/transit-signal-priority-with-spatial-info#1]
implement `log cosh` loss in pytorch
